wrapanapi/systems/ec2.py

```diff
@@ -1160,3 +1165,6 @@ def list_queues_with_creation_timestamps(self):
                 except Exception:
                     pass
         return queue_dict
+
+    def get_auth_token(self):
+        return self.ecr_connection.get_authorization_token()
```

Comment:
Please put a docblock here describing the response - as it is a dictionary with the actual token down in the structure a bit.

Consider adding some logic to check the response code in the dictionary, or to pull out and return an explicit structure with the expiration date, token, and endpoint proxy.